contracts/interfaces/IVaultAggregator.sol

```diff
@@ -29,6 +31,8 @@ interface IVaultAggregator {
     struct MasterChefWooInfos {
         uint256[] amounts;
         uint256[] rewardDebts;
+        uint256[] pendingXWooAmounts;
```

Comment:
Is it better to create a new struct `PendingAmounts` for `pendingXWooAmounts` and `pendingWooAmounts`?